submissions/OgeEjim/DOM/contact.js

```diff
@@ -0,0 +1,82 @@
+let submitForm = document.getElementById('validateForm');
+let fullName = document.getElementById('fullName');
+let email = document.getElementById('email');
+let subject = document.getElementById('subject');
+let comments = document.getElementById('comments');
+
+
+
+// function required(field,event) {
+//     if (field.value == '') {
+//         field.nextElementSibling.innerHTML = "This field is required";
+//         event.preventDefault();
+//         return false; 
+//     } else{
+//         field.nextElementSibling.innerHTML = '';
+//         return true;
+//     }
+// }
+
+function fullnameValidation(fullName,event) {
+    let fnRegex = /^[a-zA-Z]+$/;
+    if (fullName.value == "") {
+        fullName.nextElementSibling.innerHTML = "Please enter your Name";
+        event.preventDefault();
+        return false;
+    }
+     else if (!fullName.value.match(fnRegex)) {
+        fullName.nextElementSibling.innerHTML = "Only letters are allowed";
+        event.preventDefault();
+        return false;
+    } else {
+        fullName.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
```

Comment:
This `fullnameValidation` and `emailValidation`  functions look alike. Kindly write a single function that can handle both scenarios. The same goes for the last two functions